orchagent/intfsorch.cpp

```diff
@@ -289,6 +369,44 @@ void IntfsOrch::doTask(Consumer &consumer)
             {
                 vnet_name = value;
             }
+            else if (field == "mtu")
+            {
+                SWSS_LOG_ERROR("sub interface: field: mtu");
```

Comment:
why is this an error?

Reply:
Removed. It was for debugging purpose. https://github.com/Azure/sonic-swss/pull/969#discussion_r320484396